http/jsonapi/generator/internal/articles/open-api_test.go

```diff
@@ -308,16 +311,24 @@ type Service interface {
 }
 
 /*
-Router implements: Articles Test Service
+RouterWithAuthentication implements: Articles Test Service
 
 Articles Test Service
 */
-func Router(service Service) *mux.Router {
+func RouterWithAuthentication(service Service, authBackend AuthorizationBackend) *mux.Router {
+	if authBackend != nil {
+		authBackend.Init()
+	}
```

Comment:
If you remove the `Router()` method as I suggested below, you can simply assume that `authBackend` is never `nil`.